src/test/resources/com/puppycrawl/tools/checkstyle/checks/sizes/methodlength/InputMethodLengthCountEmptySmallSize.java

```diff
@@ -25,18 +25,18 @@ class AA {
         oneLine();
     }
 
-    void twoLines() { // ok
+    void twoLines() {
 
     }
 
-    void twoLines2() { // ok
+    void twoLines2() {
         oneLine();}
 
-    void twoLinesAndComment() { // ok
+    void twoLinesAndComment() {
         // some comment
     }
 
-    void oneLine() { oneLine();/* ok */ }
+    void oneLine() { oneLine();}
```

Comment:
Please revert this

Reply:
Better to keep it , or change "ok" to some other word.